nautobot/ipam/tables.py

```diff
@@ -453,7 +453,6 @@ class IPAddressTable(StatusTableMixin, RoleTableMixin, BaseTable):
     )
     tenant = TenantColumn()
     parent__namespace = tables.Column(linkify=True)
-    # TODO: add interface M2M
```

Comment:
Not needed here?

Reply:
I went ahead and moved the 4 columns (Interfaces, Devices, VM Interfaces, Virtual Machines) to the `IPAddressTable` which is used in a bunch of other detail views (Role, Prefix, VM Interface) and the bulk import/edit/delete IP Address views. This restores the behavior that was there in 1.6.